src/components/ebay-menu/component.js

```diff
@@ -118,12 +120,19 @@ module.exports = assign({}, menuUtils, {
         });
 
         scrollKeyPreventer.add(this.contentEl);
+
+        const { getIndex, destroy } = typeahead();
+        this.getIndex = getIndex;
```

Comment:
Can we call this something like: `getTypeaheadIndex` or something else? `getIndex` seems pretty generic in this case and might be a confusing name.

Reply:
Agreed.